core/scripts/local/InitializeSystem.js

```diff
@@ -13,6 +15,8 @@ const { RegistryRolesEnum } = require("../../../common/Enums.js");
 
 const argv = require("minimist")(process.argv.slice());
 
+// TODO: Need to replace openzeppelin's ERC20Mintable with our own version since we have
+// upgraded to v3
```

Comment:
Can we go ahead and do this in this PR? Is it not a drop in replacement?

Reply:
done